narwhal/primary/src/core.rs

```diff
@@ -658,29 +659,6 @@ impl Core {
         Ok(())
     }
 
-    async fn sanitize_certificate(&mut self, certificate: &Certificate) -> DagResult<()> {
-        ensure!(
-            self.committee.epoch() == certificate.epoch(),
-            DagError::InvalidEpoch {
-                expected: self.committee.epoch(),
-                received: certificate.epoch()
-            }
-        );
```

Comment:
Could you please restore this check at some point (either on the `certificate_fetcher` or on the `core`)? We do filter this on a network level, but I believe it would be good to have those in the core checks as well

Reply:
The epoch check is already part of `Certificate::verify()`, so it seems safe to remove duplicated checks. The GC check is only done in Core now. We can check GC round earlier too but the benefit seems marginal.